src/einsteinpy/symbolic/ricci.py

```diff
@@ -58,17 +63,21 @@ def from_riemann(cls, riemann):
         return cls(sympy.tensorcontraction(riemann.tensor(), (0, 2)), riemann.syms)
 
     @classmethod
-    def from_christoffels(cls, chris):
+    def from_christoffels(cls, chris, parent_metric=None):
         """
         Get Ricci Tensor calculated from Christoffel Tensor
 
         Parameters
         ----------
         chris : ~einsteinpy.symbolic.christoffel.ChristoffelSymbols
             Christoffel Tensor
+        parent_metric : ~einsteinpy.symbolic.metric.MetricTensor or None
+            Corrosponding Metric for the Ricci Tensor.
+            None if it should inherit the Parent Metric of Christoffel Symbols.
+            Defaults to None.
 
         """
-        rt = RiemannCurvatureTensor.from_christoffels(chris)
+        rt = RiemannCurvatureTensor.from_christoffels(chris, parent_metric=None)
```

Comment:
There is no point of taking `parent_metric` as argument if you are anyways gonna pass `None` to the `RiemannCuravtureTensor` class.

`rt = RiemannCuravatureTensor.from_christoffels(chris, parent_metric=parent_metric`)

Reply:
I'm going to change it as you said, but like this: 
` rt = RiemannCuravatureTensor.from_christoffels(chris, parent_metric)`
Should be right too?

Reply:
Yes, that would also work!